lib/probes/http.js

```diff
@@ -344,12 +333,13 @@ function patchServer (module, protocol) {
 
   function wrapRequestResponse (span, req, res) {
     // Report socket errors
-    req._preHandle('error', error => span.error(error))
-    res._preHandle('error', error => span.error(error))
+    req.prependListener('error', error => span.error(error));
```

Comment:
Why is this line repeated?

Reply:
one is on the `req` object and the other is on the `res` object.

Reply:
dahhh, I looked at it so many times ...